### ISSUE TITLE
Use generic `Process` from Hugo v0.119.0 for card image processing

### ISSUE THREAD

**Issue by imfing** [imfing/hextra#103]
[Hugo v0.119.0](https://github.com/gohugoio/hugo/releases/tag/v0.119.0) introduced a new function `Process` that is more general

so we don't need the if block in the `card.html` image processing logic:
https://github.com/imfing/hextra/blob/main/layouts/shortcodes/card.html#L15

**Comment by remysheppard** [comment]
Seems pretty simple. I can take a look at this tomorrow afternoon if no one has gotten to it by then.

**Comment by imfing** [comment]
yes, it's pretty neat. with that new function, the interface of the card component could be further simplified.
